lambda/lambda-router.js

```diff
@@ -0,0 +1,148 @@
+const express = require('express')
+
+const Posts = require('../data/db.js')
+
+const router = express.Router();
+
+router.use(express.json());
+
+//GET all posts
+router.get('/', async (req, res) => {
+    try {
+        const post = await Posts.find();
+        res.status(200).json(post)
+    } catch (err) {
+        console.log(err)
+        res.status(500).json({
+            message: "The posts information could not be retrieved."
+        })
+    }
+})
+
+//GET specific post
+router.get('/:id', async (req, res) => {
+    try {
+        const post = await Posts.findById(req.params.id);
+
+        if (post.length > 0) {
+            res.status(200).json(post)
```

Comment:
Great job with this check here. You could clean up the post for the user though, and get it out of the array it's in
```suggestion
            res.status(200).json(post[0])
```